critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -374,3 +375,32 @@ def test_hide_redirect(self):
             review["entity_type"], review["entity_id"]))
         redirect_url = urlparse(response.location)
         self.assertEqual(redirect_url.path, url_for('review.entity', id=review['id']))
+
+
+    def test_publish_draft_review(self):
```

Comment:
Since we are deleting data, this test should be more comprehensive IMO. Probably fetch the actual revisions before publish, assert, then fetch again after publish to confirm the correct revisions were deleted.

Reply:
Sounds Good! I've improved the tests.